build-system/tasks/helpers.js

```diff
@@ -93,6 +93,12 @@ const watchDebounceDelay = 1000;
  */
 const cache = new Map();
 
+/**
+ * Stores esbuild's watch mode rebuilders.
+ * @private @const {!Map<string, {rebuild: function():!Promise<void>}>}
+ */
+const watchedExtensions = new Map();
```

Comment:
I think this should be renamed to `watchedTargets`, since it applies to JS files and extensions. Extension building logic lives in `extension-helpers.js`, and `buildExtension()` in that file calls into `compileJs()` in this file.

(We may want to try and consolidate those two operations in future, but as things stand today, we do need a separate codepath for extension building due to the presence of CSS, cross-dependent extensions, etc.)

Reply:
Definitely should consolidate in the future :). Renaming now!